### PR TITLE
Fix links on docs page

### PR DESCRIPTION
**Pull request by hannah-radish**
- Change Resource DSL to Custom resources
- Fix broken links on docs overview page

### CODE REVIEW

**Review by miah — APPROVED** [review]

**Review by jquick — APPROVED** [review]
Thanks @hannah-radish !

**Review by jerryaldrichiii — APPROVED** [review]
Looks good to me, thanks @hannah-radish!

File: www/data/docs_sidebar.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ sidebar_links:
     link: "/docs/reference/reporters.html"
   - title: InSpec DSL
     link: "/docs/reference/dsl_inspec.html"
-  - title: Resource DSL
+  - title: Custom resources
```
Comment (kagarmoe) [inline www/data/docs_sidebar.yml]: Capitalize the R "Custom Resources"

**Review by kagarmoe — APPROVED** [review]

File: www/data/docs_sidebar.yml (hunk 1 above)
Comment (hannah-radish) [reply www/data/docs_sidebar.yml]: @kagarmoe 👍

**Pull request merged**